content/en/docs/community/release_checklist.md

```diff
@@ -126,31 +158,34 @@ git tag --sign --annotate "$RELEASE_NAME" --message "Helm release $RELEASE_NAME"
 git push upstream "$RELEASE_NAME"
 ```
 
-This new tag is going to be the base for the patch release.
+This new tag is going to be the base for the patch release. Creating a
+[helm/helm milestone](https://github.com/helm/helm/milestones) is optional for
+patch releases.
 
 Make sure to check [helm on CircleCI](https://circleci.com/gh/helm/helm) to see
-that the release passed CI before proceeding.
+that the release passed CI before proceeding. Patch releases can skip steps 2-6
+and proceed to step 7 for [PGP signature](#7-pgp-sign-the-downloads).
```

Comment:
Clarifying that if one is releasing a patch release, one can skip a number of the major/minor release steps.

Reply:
this might need to be updated if step 6 is considered for patch releases.